cypress/integration/components/Molecules/SchoolLookup.spec.js

```diff
@@ -74,51 +74,5 @@ describe('School Lookup component', () => {
         .contains('View Code')
         .as('viewCodeBtn');
     });
-
-    it('should verify schools lookup functionality', () => {
-      // Check if the API is up and running before running the tests.
-      cy.request({
-        url: 'https://lookups.sls.comicrelief.com/schools/lookup?query=NW5',
-        failOnStatusCode: false
-      })
-        .then(response => {
-          if (response.status >= 400) {
-            cy.get('@container')
-              .find('#school-lookup')
-              .type('se1 7tp')
-              .get('@container')
-              .find('[data-test="error-message"]')
-              .contains('Sorry, there was an unexpected error. Please try again');
-          } else {
-            // verify error message for non school postcode
-            cy.get('@container')
-              .find('#school-lookup')
```

Comment:
Looks like the lookup test is not right here, there are no manual fields that are displayed for `else` block to execute when lookup status >= 400 

<img width="814" alt="Screen Shot 2020-11-23 at 16 49 14" src="https://user-images.githubusercontent.com/16117926/99990869-44649780-2dac-11eb-81b2-6c6ecbe06a12.png">

Reply:
Manual fields? Sorry I don't follow

Reply:
What I mean is, when lookup status is not 200, the field itself doesn't showup for rest of the code to execute hence it fails, removing the else blocks works